content/_index.md

```diff
@@ -73,40 +69,42 @@ g3Commons:
+  para2: The Gen3 data indexing service (Indexd) assigns persistent digital IDs to data objects and returns the location of the data object in one or more public or private clouds. Indexd is compatible with the GA4GH DRS standard and is cloud-agnostic, meaning data objects can be stored in a variety of locations and even move physical locations while maintaining a single, persistent digital ID.
+  para3: Gen3 utilizes OpenID Connect for providing authentication services along with a flexible ABAC policy engine for providing tailored user authorization, depending on a data resource's specific needs. Currently supported identity providers include Google and Shibboleth via NIH iTrust, InCommon Federation, and eduGAIN. Additionally, Gen3 can interoperate with NIH dbGaP to obtain authorization information and is involved in the NIH Research Authorization Service (RAS) pilot to support RAS as an identity and authorization provider.
+  para4: The Gen3 Framework Services include logging, security, and compliance services so that Gen3 Data Commons and Gen3 Workspaces can be operated at the FISMA Moderate security compliance level.
+g3DataCommons:
+  title: Gen3 Data Commons
+  para1: Gen3 Data Commons are cyberinfrastructure that co-locates data analysis, exploration and visualization tools with data management services for import and export of structured information like clinical, phenotypic, or biospecimen data, and data objects, like genomics data files or medical images. Typically, a data model is utilized to standardize the structure of metadata input and output, although a commons may also support multiple data models by allowing submission of structured data packaged along with a custom data model, for example, Avro-based data-serialization formats like the Portable Format for Biomedical data or PFB.
+  para2: When using Gen3 to build a data commons, the platform auto-generates a data submission and exploration portal that enables the use of its features within a standard web-browser. Fully-integrated analysis workspaces can be also be included, although data can be exported to virtually any cloud-based analysis platform that allows programmatic access via Gen3 open APIs.
+  para3: Gen3 Data Commons are a kind of data resource that is interoperable with other nodes in a data ecosystem by utilizing the Gen3 Framework Services for authentication, authorization, data object indexing and metadata services.
+g3Workspaces:
+  title: Gen3 Workspaces
+  para1: A Gen3 workspace can access data from one or more data resources, including Gen3 Data Commons, and also uses the Gen3 Framework Services for user authentication and authorization and data object indexing and metadata services. Gen3 Workspaces support Jupyter notebooks, RStudio notebooks, and other applications that can access data through Gen3 open APIs.
```

Comment:
I'd suggest to remove some "ands":
"A Gen3 workspace can access data from one or more data resources, including Gen3 Data Commons, and also uses the Gen3 Framework Services for user authentication, authorization, data object indexing, and metadata services."